tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -1,510 +1,684 @@
+   "metadata": {},
+   "source": [
+    "For remote backends, the object returned is a `RemoteResults` instance, which uses the connection to fetch the results once they are ready. To check the status of the batch, we can run:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "id": "d24593f4",
+   "metadata": {
+    "tags": [
+     "skip-execution"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "BatchStatus.RUNNING\n",
+      "BatchStatus.RUNNING\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(free_results.get_batch_status())\n",
+    "print(mps_results.get_batch_status())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "763e011c",
+   "metadata": {},
+   "source": [
+    "When the batch states shows as `DONE`, the results can be accessed. They are packaged in a `RemoteResults` objects, which will contain one `Results` for each entry in `job_params` in the same order. We can use `get_available_results` to see the results as they become available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "id": "738de317",
+   "metadata": {
+    "tags": [
+     "skip-execution"
+    ]
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'bf78edf4-e25e-464d-8232-737bb210f191': Results(atom_order=('q0', 'q1'), total_duration=1000, _results={UUID('c5fa9f30-84a4-4186-96ed-70dff262671b'): [{'11': 985, '10': 7, '01': 6, '00': 2}]}, _times={UUID('c5fa9f30-84a4-4186-96ed-70dff262671b'): [1.0]}, _aggregation_methods={UUID('c5fa9f30-84a4-4186-96ed-70dff262671b'): <AggregationMethod.BAG_UNION: 3>}, _tagmap={'bitstrings': UUID('c5fa9f30-84a4-4186-96ed-70dff262671b')}),\n",
+       " 'c83b68ad-3a73-43eb-94d6-d04e120687d6': Results(atom_order=('q0', 'q1'), total_duration=2000, _results={UUID('4e272fb1-3d03-4b5c-8f25-da90977f8b37'): [{'11': 950, '01': 17, '10': 26, '00': 7}]}, _times={UUID('4e272fb1-3d03-4b5c-8f25-da90977f8b37'): [1.0]}, _aggregation_methods={UUID('4e272fb1-3d03-4b5c-8f25-da90977f8b37'): <AggregationMethod.BAG_UNION: 3>}, _tagmap={'bitstrings': UUID('4e272fb1-3d03-4b5c-8f25-da90977f8b37')})}"
+      ]
+     },
+     "execution_count": 58,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "free_results.get_available_results()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7daaec2",
+   "metadata": {},
+   "source": [
+    "`RemoteResults` store multiple types of results. You can inspect which results are present with the method get_result_tags(). For a given result type `\"my_tag\"` present in this list, `RemoteResults.my_tag` will return the list of the values observed for this type of result at each measurement time (the measurement times can be accessed in `RemoteResults.get_result_times(\"my_tag\")`). Since we used `EmuFreeBackendV2` with the default configuration, the results will contain bitstrings at the final time."
```

Comment:
```suggestion
    "`RemoteResults` store multiple types of results. You can inspect which results are present with the method `get_result_tags()`. For a given result type `\"my_tag\"` present in this list, `RemoteResults.my_tag` will return the list of the values observed for this type of result at each measurement time (the measurement times can be accessed in `RemoteResults.get_result_times(\"my_tag\")`). Since we used `EmuFreeBackendV2` with the default configuration, the results will contain bitstrings at the final time."
```